src/DGtal/geometry/doc/moduleMetrics.dox

```diff
@@ -25,218 +24,116 @@ namespace concepts{
 
+In DGtal, the concept of CMetricSpace is associated with metric space definition. A first refinement of this concept is the concept of CDigitalMetricSpace whose models implement distance functions on integer numbers (i.e. @f$(\mathbb{Z}^d, \mathbb{Z}, d)@f$)). For example,
 
-@section subConcepts Main concepts
+- @f$ l_p@f$ metrics (ExactPredicateLpSeparableMetric or  InexactPredicateLpSeparableMetric) are models of CMetricSpace for  @f$p>1@f$
+
+- Chamfer norms with axis symmetric unit ball (see experimental::ChamferNorm2D in dimension 2).
 
-In DGtal, the concept of CMetricSpace is associated with metric space
-definition. A first refinement of this concept is the concept of
-CDigitalMetricSpace whose models implement distance functions on
-integer numbers (i.e. @f$(\mathbb{Z}^n, \mathbb{Z}, d)@f$)). For
-example,
-
- - @f$ l_p@f$ metrics (ExactPredicateLpSeparableMetric or
- InexactPredicateLpSeparableMetric) are models of CMetricSpace for
- @f$p>1@f$
-
- - Chamfer norms with axis symmetric unit ball (see experimental::ChamferNorm2D in dimension 2).
-
- - DigitalMetricAdapter which adapts any metric (model of
-   CMetricSpace) to digital space using the ceil function. This induces
-   a digital metric (ceil function preserves the
-   triangular inequality).
- 
-
-@dot
-digraph metric_concepts_2_vor {
-    CMetricSpace [label="CMetricSpace" URL="@ref CMetricSpace"];
-    CDigitalMetricSpace [label="CDigitalMetricSpace" URL="@ref CDigitalMetricSpace"];
-    CSeparableMetric [label="CSeparableMetric" URL="@ref CSeparableMetric"];
-    CDigitalSeparableMetric [label="CDigitalSeparableMetric"];
-    CPowerMetric [label="CPowerMetric" URL="@ref CPowerMetric"];
-    CPowerSeparableMetric [label="CPowerSeparableMetric" URL="@ref CPowerSeparableMetric"];
-
-    CSeparableMetric -> CMetricSpace;
-    CSeparableMetric -> CMetricSpace;
-    CDigitalSeparableMetric -> CDigitalMetricSpace;
-    CDigitalSeparableMetric -> CSeparableMetric;
-    CPowerSeparableMetric -> CPowerMetric;
-
-    label="Main metric concepts";
-  }
+- DigitalMetricAdapter which adapts any metric (model of    CMetricSpace) to digital space using the ceil function. This induces    a digital metric (ceil function preserves the    triangular inequality).
+
+@dot digraph metric_concepts_2_vor {
+CMetricSpace [label="CMetricSpace" URL="@ref CMetricSpace"];
+CDigitalMetricSpace [label="CDigitalMetricSpace" URL="@ref CDigitalMetricSpace"];
+CSeparableMetric [label="CSeparableMetric" URL="@ref CSeparableMetric"];
+CDigitalSeparableMetric [label="CDigitalSeparableMetric"];
+CPowerMetric [label="CPowerMetric" URL="@ref CPowerMetric"];
+CPowerSeparableMetric [label="CPowerSeparableMetric" URL="@ref CPowerSeparableMetric"];
+
+CSeparableMetric -> CMetricSpace;
+CSeparableMetric -> CMetricSpace;
+CDigitalSeparableMetric -> CDigitalMetricSpace;
+CDigitalSeparableMetric -> CSeparableMetric;
+CPowerSeparableMetric -> CPowerMetric;
+
+label="Main metric concepts";
+}
```

Comment:
Why did you unindent these lines ? Same question about the modification below

Reply:
I didn’t in this PR

Reply:
My bad.. some weird side effects of my editor. Let me fix it